go-runtime/ftl/ftltest/fake.go

```diff
@@ -107,7 +107,7 @@ func (f *fakeFTL) setSecret(name string, value any) error {
 func (f *fakeFTL) GetSecret(ctx context.Context, name string, dest any) error {
 	data, ok := f.secretValues[name]
 	if !ok {
-		return fmt.Errorf("config value %q not found: %w", name, configuration.ErrNotFound)
+		return fmt.Errorf("config value %q not found. Did you remember to ctx := ftltest.Context(ftltest.WithDefaultProjectFile()) ?: %w", name, configuration.ErrNotFound)
```

Comment:
Generally don't use multiple sentences in Go errors, because they don't end up composing well with error chaining. eg.

```
some error: Did you remember to do X?: some other error: yet another other error
```

Instead prefer `,`, `;` or `:`, eg.:

```suggestion
		return fmt.Errorf("config value %q not found, did you remember to ctx := ftltest.Context(ftltest.WithDefaultProjectFile()) ?: %w", name, configuration.ErrNotFound)
```